NLog.Etw/NLogEtwExtendedTarget.cs

```diff
@@ -1,48 +1,45 @@
-﻿using NLog;
-using NLog.Targets;
-using System;
-using System.Diagnostics;
-using System.Diagnostics.Tracing;
+﻿using NLog.Targets;
+using Microsoft.Diagnostics.Tracing;
 
 namespace NLog.Etw
 {
     /// <summary>
-    /// A NLog target with support for extended ETW tracing. When using perfview or wpr to record the events use *LowLevelDesign-NLogEtwSource
+    /// A NLog target with support for channel-enabled ETW tracing. When using perfview or wpr to record the events use *NLog-LogEvents
     /// to enable the NLog provider.
     /// 
-    /// Sample configuration and usage sample can be found on my blog: http://lowleveldesign.wordpress.com/2014/04/18/etw-providers-for-nlog/
+    /// Channel alignment based on best practices documented here: https://blogs.msdn.microsoft.com/vancem/2012/08/14/etw-in-c-controlling-which-events-get-logged-in-an-system-diagnostics-tracing-eventsource/ 
     /// </summary>
     [Target("ExtendedEventTracing")]
     public sealed class NLogEtwExtendedTarget : TargetWithLayout
     {
-        [EventSource(Name = "LowLevelDesign-NLogEtwSource")]
```

Comment:
From [this article](https://blogs.msdn.microsoft.com/vancem/2012/08/14/etw-in-c-controlling-which-events-get-logged-in-an-system-diagnostics-tracing-eventsource/):
"Finally, the EventSource above follows a ‘best practice’ of naming its ETW provider a good name FROM THE U SER’s POINT OF VIEW.   Names should at least start with your company name.  If you expect more than one EventSource for your company it is best to give them categories separated by ‘-‘.   This is the convention that Microsoft has been following for the last several years.  (e.g. Microsoft-Windows-DotNetRuntime). "

"LowLevelDesign-xxx" doesn't seem to be a "good name from the user's point of view" (it's not obvious what that refers to).  "NLog-xxx" seems more appropriate IMHO.

It's only a breaking change with respect to guidance.  We could add updated instructions to the readme file on how to dynamically consume the events using PerfView.

Thoughts?